db/scripts/sanitise.sql

```diff
@@ -0,0 +1,154 @@
+        ELSE concat('User', id)
+        END,
+
+    phone_number = CASE
+        WHEN phone_number IS NULL THEN NULL
+        ELSE '07' || LPAD(FLOOR(RANDOM() * 100000000)::TEXT, 8, '0')
+        END,
+
+    address_line1 = CASE
+        WHEN address_line1 IS NULL THEN NULL
+        ELSE 'Address Line 1'
+        END,
+
+    address_line2 = CASE
+        WHEN address_line2 IS NULL THEN NULL
+        ELSE 'Address Line 2'
+        END,
+
+    address_line3 = CASE
+        WHEN address_line3 IS NULL THEN NULL
+        ELSE 'Address Line 3'
+        END,
+
+    address_line4 = CASE
+        WHEN address_line4 IS NULL THEN NULL
+        ELSE 'Address Line 4'
+        END,
+
+    postcode = CASE
+        WHEN postcode IS NULL THEN NULL
+        ELSE 'SW1P 3BT'
+        END,
+
+    disability_disclosure = CASE
+        WHEN  disability_disclosure IS NULL THEN NULL
+        ELSE  generate_lorem_ipsum('short')
+        END,
+
+    becoming_a_teacher = CASE
+        WHEN becoming_a_teacher IS NULL THEN NULL
+        ELSE generate_lorem_ipsum('long')
+        END,
+
+    safeguarding_issues = CASE
+        WHEN safeguarding_issues IS NULL THEN NULL
+        ELSE generate_lorem_ipsum('short')
+        END,
+
+    international_address = CASE
+        WHEN international_address IS NULL THEN NULL
+        ELSE '20 W 34th St., New York, NY 10001, USA'
+        END,
+
+    right_to_work_or_study_details = CASE
+        WHEN right_to_work_or_study IS NULL THEN NULL
+        ELSE generate_lorem_ipsum('short')
+        END;
+
+-- ApplicationChoice
+UPDATE "application_choices"
+SET personal_statement = CASE
+    WHEN  personal_statement IS NULL THEN NULL
+    ELSE generate_lorem_ipsum('long')
+    END;
+
+-- Candidate
+UPDATE "candidates"
+SET
+    email_address = concat('candidate_user_', id, '@example.com')
+WHERE email_address NOT LIKE '%@digital.education.gov.uk'
+  AND email_address NOT LIKE '%@education.gov.uk';
+
+-- ProviderUser
+UPDATE "provider_users"
+SET
+    email_address = concat('provider_user_', id, '@example.com'),
+    first_name = 'Provider',
+    last_name = concat('User', id)
+
+WHERE email_address NOT LIKE '%@digital.education.gov.uk'
+  AND email_address NOT LIKE '%@education.gov.uk';
+
+-- Reference
+UPDATE "references"
+SET
+    email_address = concat('application_reference_', id, '@example.com'),
+    feedback = CASE
+        WHEN feedback IS NULL THEN NULL
+        ELSE generate_lorem_ipsum('short')
+        END,
+    name = concat('Application Reference_', id)
+
+WHERE email_address NOT LIKE '%@digital.education.gov.uk'
+  AND email_address NOT LIKE '%@education.gov.uk';
+
+-- SupportUser
+UPDATE "support_users"
+SET
+    email_address = concat('application_reference_', id, '@example.com'),
+    first_name  = 'Support',
+    last_name = concat('User', id)
+
+WHERE email_address NOT LIKE '%@digital.education.gov.uk'
+  AND email_address NOT LIKE '%@education.gov.uk';
+
+-- VendorApiUser
+UPDATE "vendor_api_users"
```

Comment:
This makes me think we should change the AuthenticationTokens table, and possibly the VendorAPITokens. 
Not sure about the VendorAPIRequests 🤔